### ISSUE TITLE
fairseq-train: error: unrecognized arguments: --min-lr 1e-09

### ISSUE THREAD

**Issue by YingdaFan** [pytorch/fairseq#3878]
### Before asking:   
1. search the issues.   
2. search the docs.    

#### What is your question?
https://github.com/yulu-dada/Attention-calibration-NMT

run the following command and reported bug 'fairseq-train: error: unrecognized arguments: --min-lr 1e-09'

#### Code
user_dir=./mycode-gate/
data_bin=/home/ydfan/fairseq/data-bin/1
model_dir=./models/chen-gate-0.03
export CUDA_VISIBLE_DEVICES=3
fairseq-train $data_bin \
        --user-dir $user_dir --criterion my_label_smoothed_cross_entropy --task attack_translation_task --arch my_arch \
        --optimizer myadam --adam-betas '(0.9, 0.98)' --clip-norm 0.0 --lr-scheduler inverse_sqrt \
        --warmup-init-lr 1e-07 --warmup-updates 4000 --lr 0.0005 --min-lr 1e-09 \
        --weight-decay 0.0 --label-smoothing 0.1 \
        --max-tokens 2048 --no-progress-bar --max-update 150000 \
        --log-interval 100 --save-interval-updates 1000 --keep-interval-updates 10 --save-interval 10000 --seed 1111 \
        --ddp-backend no_c10d \
        --dropout 0.3 \
        -s zh -t en --save-dir $model_dir \
        --mask-loss-weight 0.03

#### What's your environment?

 - fairseq Version: 1.0
 - PyTorch Version :1.8.1
 - OS:  Linux
 - How you installed fairseq: git clone
 - Build command you used (if compiling from source):
 -
 - Python version:3.8.8
 - CUDA/cuDNN version:10.1
 - GPU models and configuration:NVIDIA-SMI 430.64       Driver Version: 430.64

**Comment by YingdaFan** [comment]
Imcompatible Fairseq Version.
